docs/pages/guides/analyzing-bundles.mdx

```diff
@@ -105,11 +105,12 @@ Web bundles are output to the **dist/client** subdirectory to prevent exposing s
 
 <Step label="3">
 
-Export your production JavaScript bundle and include the `--source-maps` flag so that the source map explorer can read the source maps:
+Export your production JavaScript bundle and include the `--source-maps` flag so that the source map explorer can read the source maps. For native apps using hermes, you can use `--no bytecode` option to disable bytecode generation.
```

Comment:
```suggestion
Export your production JavaScript bundle and include the `--source-maps` flag so that the source map explorer can read the source maps. For native apps using hermes, you can use `--no-bytecode` option to disable bytecode generation.
```